core/market/decentralized/src/db/model/subscription_id.rs

```diff
@@ -173,6 +174,25 @@ impl From<SubscriptionParseError> for ErrorMessage {
     }
 }
 
+/// Pretty display vector, as list with all elements in separate row.
+/// Implementation uses Display trait for all elements (not Debug) to make
+/// them more readable.
+pub struct DisplayVec<'a, T>(pub &'a Vec<T>);
+
+impl<'a, T> Display for DisplayVec<'a, T>
+where
+    T: Display,
+{
+    fn fmt(&self, f: &mut Formatter) -> Result<(), Error> {
```

Comment:
there is https://docs.rs/itertools/0.8.2/itertools/trait.Itertools.html#method.format_with which is already used by our deps. so adding this dep will not increase resulting binary size most probably

Reply:
I used this formatter within `DisplayVec` implementation. If you meant to use it not here, but in places I display this vector, I'm not sure it would be a good idea, because it seems to be to long

Reply:
That's what I ment. Thx